### ISSUE TITLE
When closing the share modal window

### ISSUE THREAD

**Issue by yeonkim** [Esri/geoform-template-js#504]
When a form is submitted, we get a share modal window. Clicking' Close' button will take users back to the blank form. Is there a way to take users to a specific URL (in my case, back to the main website launching page) when 'Close' button is clicked?

(I searched issue page but couldn't find similar Q & A. Sorry if this is a duplicate question.)

Thanks!
Yeon

**Comment by driskull** [comment]
There's not an option for that currently. Its something that could be added as an option pretty easily though.

**Comment by yeonkim** [comment]
Can you point me where  I should make changes to make this happen? :)

It would be also great if this can added as a configurable option. Going back to a blank form after the submission can be confusing, esp if the form is designed to gather feedback only once per user.

Thanks again.
Yeon

**Comment by driskull** [comment]
yeah, I would put something in the `resetAndShare` function here:

https://github.com/Esri/geoform-template-js/blob/master/js/main.js#L2234

Once the form has been submitted and attachments uploaded, this function gets called.

You could add an option in the config like "submitRedirectUrl" and check to see if it has a value. If it does have a value, instead of putting the form back to the top, just redirect to that URL.

**Comment by yeonkim** [comment]
Matt,

Thanks for showing me the direction. :)

As you suggested, I added the config "submitRedirectURL" in default.js. in main.js, I added the following for modal closing event in `openShareModal`

https://github.com/Esri/geoform-template-js/blob/master/js/main.js#L2425

`	   $('#myModal').on('hidden.bs.modal', lang.hitch(this, function () {
		  if (this.config.submitRedirectUrl != '') {
			window.location.href = this.config.submitRedirectUrl;
		   }
      }));`

`resetAndShare` did not offer  the right  timing to redirect the page. 

I wouldn't have been able to figure this tweak out without your pointers. Thanks again!!

Yeon
